.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build:
+    if: github.repository == 'typeddjango/django-stubs'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U setuptools twine wheel
+
+      - name: Build package
+        run: |
+          python setup.py --version
+          python setup.py sdist bdist_wheel
+          twine check dist/*
+
+      - name: Publish package to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+          print_hash: true
+          verify_metadata: false  # check done above
+
+      - name: Create release
```

Comment:
Let's create an issue if this fails. https://github.com/sobolevn/git-secret/blob/master/.github/workflows/release-ci.yml#L73-L91

Reply:
Done, not sure it's as needed because the example you gave is only being triggered on `schedule`, but it doesn't seem like it would hurt.